addon-test-support/@ember/test-helpers/dom/fire-event.js

```diff
@@ -180,8 +180,9 @@ function buildKeyboardEvent(type, options = {}) {
 }
 
 // eslint-disable-next-line require-jsdoc
-function buildFileEvent(type, element, files = []) {
+function buildFileEvent(type, element, options = {}) {
   let event = buildBasicEvent(type);
+  let files = Array.isArray(options) ? options : options.files || [];
```

Comment:
Should we issue a deprecation or something when we do this coercion?

I'm just thinking that over time we _probably_ only want to support one form...